src/components/PositionsList/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Pagination } from '@material-ui/lab'
+import useStyle from './style'
+import { useWindowWidth } from '@react-hook/window-size'
+export interface IProps {
+  pages: number
+  defaultPage: number
+  handleChangePage: (event: React.ChangeEvent<unknown>, page: number) => void
+}
+
+export const PaginationList: React.FC<IProps> = ({ pages, defaultPage, handleChangePage }) => {
+  const classes = useStyle()
+  const onlyWidth = useWindowWidth()
```

Comment:
you don't need additional libraries here, just use "useMediaQuery" hook from material-ui, there should be some examples in syntetify/invariant repos